### PR TITLE
Change -H flag behaviour to show only hash

### PR DESCRIPTION
**Pull request by aqk**

### CODE REVIEW

File: tests/cmds_test.py [hunk 1]
```diff
@@ -101,6 +101,8 @@ def inject(*paths):
 
 inject("opc")
 
+inject("opd")
```
Comment (arvidn) [inline tests/cmds_test.py:104]: were those tests not run before?

File: tests/opd/hash.txt [hunk 2]
```diff
@@ -1,3 +1,2 @@
 opd -H ff0109
-9c2522e2c34f46724c6a4e95f460ced3eb30b72eb6b19a8cc832681fd75aabab
```
Comment (arvidn) [inline tests/opd/hash.txt:2]: how come the hash changed here?

File: tests/cmds_test.py (hunk 1 above)
Comment (aqk) [reply tests/cmds_test.py:104]: Correct.

File: tests/opd/hash.txt (hunk 2 above)
Comment (aqk) [reply tests/opd/hash.txt:2]: Good question :)
This is because we updated `opd -H` to use `sha256tree` instead of `sha256(bytes(prog))`

**Review by richardkiss — APPROVED** [review]

**Pull request merged**